test/metabase/test/data.clj

```diff
@@ -247,3 +250,18 @@
   {:style/indent 0}
   [& body]
   `(data.impl/do-with-temp-copy-of-db (fn [] ~@body)))
+
+(defmacro with-empty-h2-app-db
```

Comment:
moved it here.

Reply:
Can you remove it from the `metabase-enterprise.serialization.test-util` namespace?

Reply:
removed!